docs/config.rst

```diff
@@ -1086,8 +1086,22 @@ user can bypass the system's interpreter parser by defining the
 When the workaround is enabled, all tox-invoked text file executables will have
 their interpreter directive parsed by and explicitly executed by ``tox``.
 
+Environment variables
+---------------------
+tox will treat the following environment variables:
```

Comment:
I think you need either a double colon or an empty line, so the list gets properly rendered - have a look at https://github.com/tox-dev/tox/blob/e28a4983c59c1470f8a4d9ee482d28157bd83bbb/docs/config.rst#environment-variables - currently this does not look good.

You can also build the docs locally on your machine before pushing via `tox -e docs`